### ISSUE TITLE
excessive regrid warning 

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/acme_diags#142]
We shouldn't have this warning since regridTool is set to esmf as default. 

```
/export_2/zhang40/anaconda2/envs/acme_diags_env/lib/python2.7/site-packages/cdms2/avariable.py:1142: Warning: 
avariable.regrid: We chose regridTool = esmf for you among the following choices:
   Tools ->    'regrid2' (old behavior)
               'esmf' (conserve, patch, linear) or
               'libcf' (linear)
  warnings.warn(message, Warning)
```

**Comment by gleckler1** [comment]
@chengzhuzhang @dnadeau4  Is the default "conservative" or "linear"?   I was just looking at Curt Covey's code this morning and assumed it was using "regrid2"... guess I was wrong.
 
 g = cdms2.open('dailyMean034Jul1999-2005std_of_dailymeans.nc')
   v = g['dailySD'] # Randomly chosen variable on CESM-LE grid
   outgrid = v.getGrid()
   print 'Target-grid shape:', outgrid.shape
   tvarbnew = tvarb.regrid(outgrid)

**Comment by dnadeau4** [comment]
- The default is `tool="libcf"`,  `method="linear"` when data has no bounds defined.
- The default is `tool="esmf"`,  `method="linear"` when data has bounds.
- The default is `esmf` if `method=conservative` or `patch`.
- The default for non-uniform grids, tool=`esmf`, method=`linear`

**Comment by gleckler1** [comment]
@dnadeau4 Thanks... good to know!

**Comment by chengzhuzhang** [comment]
@dnadeau4 we used 'regridTool' and regrid_tool  = 'esmf'. it doesn't seem like an misspelling...
mv2_reg = mv2.regrid(mv_grid, regridTool=regrid_tool,
                             regridMethod=regrid_method)

**Comment by chengzhuzhang** [comment]
@dnadeau4 I didn't put the keyword 'regridTool=' and 'regridMethod=' in one regrid line, this triggered the warning messages...This is fixed in commit https://github.com/ACME-Climate/acme_diags/commit/25e01721911ad0896802ee7d14bab2904a36f47e
Thank you for troubleshooting with us!

**Comment by chengzhuzhang** [comment]
@dnadeau4 sorry to bother you one more time...if 'conserve' (this is in the warning message) or 'conservative' both okay for conservative regrid regard method?

**Comment by zshaheen** [comment]
We had the arguments mixed up, where the regrid tool and regrid method were swapped. Hence, that's why it defaulted to `esmf` with `linear` regridding.

Everything is fixed in 25e01721911ad0896802ee7d14bab2904a36f47e.
